src/main/java/com/microsoft/graph/requests/GraphClientOption.java

```diff
@@ -97,6 +96,6 @@ public <T extends RequestOption> Class<T> getType() {
     }
 
     private static String paramErrorMessage(String paramName){
-        return ("Parameter "+paramName+" cannot be null.");
+        return ("Parameter "+paramName+ "cannot be null.");
```

Comment:
nit: Error message will be malformed with the removed space. The `paramName` will be joined to to `cannot`